.github/workflows/nightly-trivy-scan.yml

```diff
@@ -0,0 +1,37 @@
+name: Trivy Nightly Scan
+on:
+  schedule:
+    - cron: '0 2 * * *' # run at 2 AM UTC
+
+
+jobs:
+  nightly-scan:
+    name: Trivy Scan nightly
+    strategy:
+      fail-fast: false
+      matrix:
+        # maintain the versions of harbor that need to be actively
+        # security scanned
+        versions: [dev, v2.6.2, v1.10.14]
```

Comment:
let's use the `version: [dev]` instead

Reply:
In that case, how will we identify CVEs that are reported in older active versions of harbor

Reply:
How about we keep dev for regular default scanning, and when we have 2.7 release, we will add the latest  release branch/tag for scanning?

Reply:
Updated